functional-programming/combine-an-array-into-a-string-using-the-join-method.js

```diff
@@ -0,0 +1,8 @@
+function sentensify(str) {
+// Only change code below this line
+let modifiedStr = str.split(/\W/);
```

Comment:
Change the name of this variable. When you apply split method the return is an array and doesn't make sense you call this variable modifiedStr because it's not a string. 
As I've said before when you split a phrase, what you have? -> words
Other good practise is don't put the data structure into variable name...
It's an array, don't call the variable namesArray use names
It's an age formatted as string, don't use ageStr, use age